src/js/components/Button/stories/Busy.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Box, Button } from 'grommet';
+
+export const Busy = () => {
+  const [busy, setBusy] = useState();
+
+  return (
+    <Box align="center" pad="medium">
+      <Button
+        primary
+        busy={{
+          state: busy,
```

Comment:
Would it make more sense for `busy` to be a boolean. Then the "success" state would automatically rendered for "X" amount of time when busy changes from true to false?

Reply:
Pausing to think about what happens if a button is busy, then the process fails..

Reply:
Based on our discussion yesterday evening I changed `busy` to be a boolean. I changed `messages` to no longer be nested within busy and instead just a prop on Button. I also added a `success` prop that is a boolean.